### PR TITLE
Fix send info in AlertManever

### PR DESCRIPTION
**Pull request by OHerasym**
Fix for AlertManever response to mobile. AlertManaver sends wrong "info" parameter to mobile.
Related:
        [APPLINK-23121](https://adc.luxoft.com/jira/browse/APPLINK-23121)

### CODE REVIEW

**Comment by OHerasym** [comment]
@AByzhynar, @AGaliuzov , @Kozoriz, @LuxoftAKutsan , @anosach-luxoft , @dtrunov , Please review

File: src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc [hunk 1]
```diff
@@ -135,7 +135,8 @@ void AlertManeuverRequest::on_event(const event_engine::Event& event) {
   LOG4CXX_AUTO_TRACE(logger_);
   using namespace helpers;
   const smart_objects::SmartObject& message = event.smart_object();
-
+  bool is_tts_ok, is_no_navi_error;
+  hmi_apis::Common_Result::eType tts_result, navi_result;
```
Comment (AByzhynar) [inline src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc]: @OHerasym Every variable must be declared on separate line. Even of the same type.

**Comment by AByzhynar** [comment]
Reviewed

**Comment by OHerasym** [comment]
@AByzhynar , 632e99a, Done

**Comment by dtrunov** [comment]
Reviewed

**Comment by Kozoriz** [comment]
Revieved

**Comment by anosach-luxoft** [comment]
Reviewed. @OHerasym please create task to refactor this request (simplify `on_event` method to avoid all these unclear variables usage)

File: src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc [hunk 2]
```diff
@@ -160,6 +178,24 @@ void AlertManeuverRequest::on_event(const event_engine::Event& event) {
...
+      is_tts_ok = Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
+          tts_result,
+          hmi_apis::Common_Result::SUCCESS,
+          hmi_apis::Common_Result::UNSUPPORTED_RESOURCE,
+          hmi_apis::Common_Result::WARNINGS,
+          hmi_apis::Common_Result::INVALID_ENUM);
+
+      const bool is_tts_success =
+          Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
+              tts_result, hmi_apis::Common_Result::SUCCESS);
```
Comment (AGaliuzov) [inline src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc]: I think it is excessive to use `Compare` for only one parameter. It is better to use direc equal operator

**Comment by AGaliuzov** [comment]
Reviewed

**Pull request merged**